src/content/dataroom.json

```
@@ -0,0 +1,134 @@
+[
+  {
+    "pageTitle": "Safe Data Room - Key Insights into the Safe Ecosystem",
+    "description": "Uncover the impact of the Safe ecosystem in crypto. Explore detailed metrics on USDC storage, CryptoPunks holdings, and transaction volumes. Access comprehensive reports from Dune Analytics, Messari, Nansen, and more.",
+    "image": "/images/Home/meta-image-homepage.jpg",
+    "component": "common/MetaTags"
+  },
+  {
+    "component": "DataRoom/Hero",
+    "title": "Welcome to Safe Data Room",
+    "text": "Explore Mind Bending Sats from Safe Universe"
+  },
+  {
+    "component": "DataRoom/USDCStorage",
+    "title": "We store... <br/>",
```

Do we need the line break here?

```suggestion
    "title": "We store...<br/>",
```